### DISCUSSION TITLE
Cannot see value if it has only one groupby field, works perfectly if the fileds are more than 1 (>1)

### DISCUSSION THREAD

**Discussion by yogeshboddu** [grafana/grafana-json-datasource#324] [Q&A]
![image](https://user-images.githubusercontent.com/95027376/211580859-45980761-3b07-4aac-80a3-b873fab9548f.png)
![image](https://user-images.githubusercontent.com/95027376/211581924-f153a895-ad85-4c3e-802a-14d5c34511c6.png)

**Comment by yesoreyeram** [comment]
can you provide examples of what your data looks like and what your query looks like? ( also note that group by is an experimental feature )

**Reply by yogeshboddu** [reply]
1.  If I have json as 

```
[      
        {
		"total": 20,
		"repo": "val1"
	}, 
        {
		"total": 30,
		"repo": "val1"
	},
	{
		"total": 12,
		"repo": "val2"
	}
]
```
we have 2 distinct repo values( > 1 ) and use groupby repo(total-aggregate) Bargauge
I can see both values (val1, val2) on Y-Axis

2. But If I have values as below

```
[
   {
	"total": 20,
	"repo": "val1"
   },
   {
	"total": 30,
	"repo": "val1"
   }
]
```
We have distinct repo value is 1 and use groupby repo(total-aggregate) Bargauge
can't see any values on Y-Axis 😔

I have experienced this problem in Bargauge, Bargraph and state timeline

**Comment by yogeshboddu** [comment]
@yesoreyeram can you please help in fixing this bug
